src/seedu/addressbook/data/person/Person.java

```diff
@@ -81,7 +81,18 @@ public int hashCode() {
 
     @Override
     public String toString() {
-        return getAsTextShowAll();
+       return getPrintableString(phone,email,address);
+    }
+
+    /**
+     * Returns a concatenated version of the printable strings of each object.
+     */
+    String getPrintableString(Printable... printables){
```

Comment:
Ideally, `getPrintableString(Printable... printables)` should be used somewhere.